curl-sys/build.rs

```diff
@@ -39,6 +39,8 @@ fn main() {
         if windows {
             if try_vcpkg() {
                 return;
+            } else if try_pkg_config() {
```

Comment:
I'd only run `try_pkg_config` when targeting non-MSVC targets.

Reply:
Actually, on second thought, this is fine; will explain in a comment on this PR.